src/libs/ck-libs/armci/armci_vp.C

```diff
@@ -195,7 +195,7 @@ void ArmciVirtualProcessor::get(pointer src, pointer dst,
   }*/
   thisProxy[src_proc].requestFromGet(src, dst, nbytes, thisIndex, -1);
   // wait for reply
-  thread->suspend();
+  TCharm * unused = thread->suspend();
```

Comment:
Won't this cause lots of "variable set but not used" warnings?

Reply:
I don't believe we have that warning enabled, or else we could get tons of other instances of it. I named the variable `unused` because I _think_ it will silence the warning at least on some systems but I'm not sure.

Reply:
Eventually we can add C++17's [[maybe_unused]] attribute here and throughout the runtime